framework/src/modules/chain/logic/transaction_pool.js

```diff
@@ -366,6 +366,10 @@ class TransactionPool {
 			]);
 		}
 
+		if (transaction.bundled) {
```

Comment:
Can we have some tests for this? Looks like the presence of this flag is changing the way of how the transaction is processed and to which queue is added in the transaction pool.